src/main/java/com/nextcloud/common/NextcloudClient.kt

```diff
@@ -50,40 +47,47 @@ import javax.net.ssl.SSLSession
 import javax.net.ssl.TrustManager
 
 
-class NextcloudClient(var baseUri: Uri, val context: Context) {
+class NextcloudClient(var baseUri: Uri, val context: Context, client: OkHttpClient) {
     lateinit var credentials: String
```

Comment:
Misused `lateinit`. This should be initialized here. I'll open another ticket for this, tho.

Reply:
This is fixed via #396